tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -942,6 +965,73 @@ void ReorderRSCKToKCRS(const TRT_ShapedWeights& iweights,
   }
 }
 
+nvinfer1::Dims InitDims5(const int a, const int b, const int c, const int d,
```

Comment:
Can we use std::initializer_list<int> and let kNUM_DIMS=list.size() to make it more generic?

Reply:
Yes, good call